util/path_debug.go

```diff
@@ -3,7 +3,8 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-//+build debug
+//go:build debug
+// +build debug
```

Comment:
Is this change needed? In my setup, lint successfully completes without this change.

Reply:
This change was done by `go fmt ./...`, it was not a manual change

Reply:
Ok